icu4c/source/common/wintz.cpp

```diff
@@ -13,14 +13,12 @@
 
 #include "unicode/utypes.h"
 
-// This file contains only desktop Windows behavior
```

Comment:
This file is currently excluded from the UWP project file for the `common` project. If the plan is to replace the Windows.Globalization code with this API call (which is goodness), then you will need to modify the `common_uwp.vcxproj` file too.

Reply:
The file is updated in the latest change.